cpp/harvest_ukdataservice.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Can we move all of these harvesting-related programs into a new subfolder? like e.g. `cpp/harvest` 
This would also affect programs like ch.sh, cologne.sh, the whole bnb_yaz stuff and maybe more.
(Of course, this would imply modifying cronjobs + Makefiles + maybe system_updates as well)

Reply:
... as i thought about this a bit more, ch.sh and cologne.sh is not data we harvest, but data we provide to others. Should we group those scripts into a seperate subfolder as well?

Reply:
We now have ../cpp/data_harvests.